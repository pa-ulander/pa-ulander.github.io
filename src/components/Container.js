import React from 'react'
import Header from './Header'

const Container = ({ children }) => {
  return (
    <div>
      <div className="container">
        <Header />
        {children}
      </div>
    </div>
  )
}

export default Container
