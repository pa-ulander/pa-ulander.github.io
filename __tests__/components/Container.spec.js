import React from 'react'
import { shallow } from 'enzyme'
import Container from '../../src/components/Container'

describe('Container component', () => {
  it('should render correctly', () => {
    const component = shallow(<Container />)
    expect(component).toMatchSnapshot()
  })
})
