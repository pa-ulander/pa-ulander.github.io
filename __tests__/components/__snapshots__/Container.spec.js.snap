// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component should render correctly 1`] = `
<div>
  <div
    className="container"
  >
    <Header />
    <main
      className="main"
    />
  </div>
</div>
`;
